src/cordova/build.action.mjs

```diff
@@ -33,35 +35,46 @@ export async function main(...parameters) {
   await runAction('cordova/setup', ...parameters);
 
   if (buildMode === 'debug') {
-    console.warn(`WARNING: building "${cordovaPlatform}" in [DEBUG] mode. Do not publish this build!!`);
+    console.warn(`WARNING: building "${outlinePlatform}" in [DEBUG] mode. Do not publish this build!!`);
   }
 
-  if (outlinePlatform === 'macos' || outlinePlatform === 'ios') {
-    const WORKSPACE_PATH = `${process.env.ROOT_DIR}/src/cordova/apple/${outlinePlatform}.xcworkspace`;
-    // TODO(fortuna): Specify the -destination parameter for build. Do we need it for archive?
+  if (cordovaPlatform === 'osx' || cordovaPlatform === 'ios') {
+    const xcodebuildBaseArguments = [
+      'xcodebuild',
+      'clean',
+      '-workspace',
+      path.join(getRootDir(), 'src', 'cordova', 'apple', `${outlinePlatform}.xcworkspace`),
+      '-scheme',
+      'Outline',
+    ];
+
     if (buildMode === 'release') {
-      execSync(`xcodebuild -workspace ${WORKSPACE_PATH} -scheme Outline -configuration Release clean archive`, {
-        stdio: 'inherit',
-      });
-    } else {
-      execSync(
-        `xcodebuild -workspace ${WORKSPACE_PATH} -scheme Outline -configuration Debug build CODE_SIGN_IDENTITY="" CODE_SIGNING_ALLOWED=NO`,
-        {
-          stdio: 'inherit',
-        }
-      );
+      return spawnStream(...xcodebuildBaseArguments, 'archive', '-configuration', 'Release');
     }
-    return;
+
+    // TODO(fortuna): Specify the -destination parameter for build. Do we need it for archive?
```

Comment:
This is what I see in the build output:
![image](https://user-images.githubusercontent.com/113565/222830626-8b6f253d-23f4-4c71-8c4f-17016f4b0e4b.png)

That's because we don't specify the destination.
No need to fix in this PR, but I think a destination like this would fix the warning:
- iOS: `platform=iOS,name=Any iOS Device`
- macOS: `platform=macOS,name=Any Mac`

Reply:
For reference:
![image](https://user-images.githubusercontent.com/113565/222837895-fbc6cd14-7730-44c5-9a35-393519706fe0.png)

That's the library test, not the app build. The "Any" suggestion if for the build only. You need to specify a concrete platform for the test.

Reply:
Awesome!